### PR TITLE
defensive programming around use of navigationBarLabel property

### PR DESCRIPTION
**Pull request by bell87**
Fixes issue with infinite loading spinner when `navigationBarLabel` global property isn't set

### CODE REVIEW

File: js/adapt-languagePicker.js [hunk 1]
```diff
@@ -60,10 +60,16 @@ define([
     }
     
     function setupNavigationView () {
+        var courseGlobals = Adapt.course.get('_globals')._extensions;
+        var navigationBarLabel = 'Select course language';
```
Comment (oliverfoster) [inline js/adapt-languagePicker.js]: Hard-coded strings aren't a good idea. Could you make the ```"aria-label"``` a null string if it's not available in the _globals object?
I'm not entirely sure why it wouldn't be available. It comes through from the schema and the AT. What's the use-case for this?

File: js/adapt-languagePicker.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ define([
     
     function setupNavigationView () {
         var courseGlobals = Adapt.course.get('_globals')._extensions;
-        var navigationBarLabel = 'Select course language';
+        var navigationBarLabel = '';
         if (_.has(courseGlobals, '_languagePicker')) {
```
Comment (oliverfoster) [inline js/adapt-languagePicker.js]: Just as a small note.  
```javascript
if (_.has(courseGlobals, '_languagePicker')) {
``` 
can be written as
```javascript
if (courseGlobals._languagePicker) {
```
which avoids a whole lot of extra computation for making the ``"_languagePicker"`` string, looking up the ``has`` property on the ``_`` object, executing the ``has`` function, making the function closure, do all the extra comparisons and return:

```javascript
_.has = function(obj, key) {
    return obj != null && hasOwnProperty.call(obj, key);
};
```

**Comment by oliverfoster** [comment]
i still don't get the use-case for this.

it will always be included from the schema unless otherwise manually deleted:

https://github.com/adaptlearning/adapt-contrib-languagePicker/blob/master/properties.schema#L7

**Comment by brian-learningpool** [comment]
On the framework this isn't an issue, but courses built in the authoring tool before the ARIA label was added will break unless the extension is removed then added again.  

This PR corrects that.

**Comment by oliverfoster** [comment]
this needs to be fixed in the AT really

**Pull request merged**